l10n_it_fatturapa_pec/models/fatturapa_attachment_out.py

```diff
@@ -97,6 +97,14 @@ def parse_pec_response(self, message_dict):
         attachments = [x for x in message_dict['attachments']
                        if regex.match(x.fname)]
 
+        if not attachments:
+            raise UserError(_(
+                    "PEC message \"%s\" is coming from SDI but attachments do not "
+                    "match SDI response format. Please check."
+                ) % (
+                    message_dict['subject']
+                ))
+
```

Comment:
Questa funzione viene eseguita dall'utente per cui vede l'errore? O è in background e compare l'errore a random?

Reply:
@sergiocorato Viene eseguita tipicamente dal cron di fetchmail, per cui, dopo `fetchmail.pec.max.retry`, l'utente viene notificato

Reply:
Questo messaggio quindi è visibile nel log e poi l'utente viene notificato dopo le max retry del fallimento della lettura dei messaggi?

Reply:
Esatto, vedi https://github.com/OCA/l10n-italy/blob/c679827d9286971eec5400382c59714975d5cb5b/l10n_it_fatturapa_pec/models/fetchmail.py#L69

Reply:
Ok